codeforsanjose/open-disclosure
# Scraper now works and is scraping as before

# How to Test
- Create a virtual environment in Python. 
```python -m virtualenv venv```
- Source the environment
```source venv/Scripts/activate```
- Install the new dependencies
```python -m pip install -r requirements.txt```
- Test the scraper
```python data_pipeline/scraper/scraper.py```
- See ```./aggregated_data/data.csv```
